e2e/cypress/tests/support/ui/channel_header.d.ts

```diff
@@ -81,6 +81,6 @@ declare namespace Cypress {
          * @example
          *   cy.uiOpenChannelMenu();
          */
-        uiOpenChannelMenu(menu?: string): Chainable;
+        uiOpenChannelMenu(item?: string): Chainable;
```

Comment:
Please update to latest and revert change here.

Reply:
Reverted and fixed the JSDoc in b1cbaae6cf563bbeb3070efb22f97a05c8787855